config/webpack.config.js

```diff
@@ -25,14 +25,7 @@ module.exports = {
         extensions: paths.moduleFileExtensions,
     },
     entry: paths.projEntry,
-    externals: [
```

Comment:
We _may_ still want to include `@geocortex` in here for Workflow sake?

Reply:
Good point. As per our discussion, I recently removed all references to @geocortex/workflow from Web's framework, and will try to keep it that way moving forward.